packages/clerk-js/src/ui/customizables/__tests__/parseAppearance.test.tsx

```diff
@@ -233,6 +233,7 @@ describe('AppearanceProvider layout flows', () => {
           layout: {
             helpPageUrl: 'test',
             logoImageUrl: 'test',
+            logoLinkUrl: 'test',
```

Comment:
@nikosdouvlis @desiprisg Not related to this PR, but it would be nice to have some actual test URLs here.

Reply:
Yup makes sense - adding a ticket